blocks/author-bio-block/features/author-bio/default.jsx

```diff
@@ -1,54 +1,30 @@
 import React, { Fragment } from "react";
 import PropTypes from "@arc-fusion/prop-types";
 import { useFusionContext } from "fusion:context";
-import getThemeStyle from "fusion:themes";
 import getTranslatedPhrases from "fusion:intl";
-import styled from "styled-components";
-import { LinkSVGHover } from "@wpmedia/news-theme-css/js/styled/linkHovers";
+import { LazyLoad } from "@wpmedia/engine-theme-sdk";
 import {
+	formatSocialURL,
+	Icon,
 	Image,
-	EnvelopeIcon,
-	InstagramIcon,
-	LinkedInIcon,
-	TwitterIcon,
-	FacebookIcon,
-	RedditIcon,
-	YoutubeIcon,
-	MediumIcon,
-	TumblrIcon,
-	PinterestIcon,
-	SnapchatIcon,
-	WhatsAppIcon,
-	SoundCloudIcon,
-	RssIcon,
-	LazyLoad,
 	isServerSide,
-	constructSocialURL,
-} from "@wpmedia/engine-theme-sdk";
-import { PrimaryFont } from "@wpmedia/shared-styles";
+	Stack,
+	Paragraph,
+	Link,
+} from "@wpmedia/arc-themes-components";
 import getProperties from "fusion:properties";
 
-import "./author-bio.scss";
-import "@wpmedia/shared-styles/scss/_author-bio.scss";
+const BLOCK_CLASS_NAME = "b-author-bio";
 
-const MediaLinksStyled = styled(LinkSVGHover)``;
-
-const renderAuthorInfo = (author, arcSite) => {
+const renderAuthorImage = (author, arcSite) => {
 	const { image = {}, name, resized_params: resizedImageOptions } = author;
 
 	const { url = "", alt_text: altText = "" } = image;
 
 	return url ? (
 		<Image
-			url={url}
+			src={url}
 			alt={altText || name}
-			smallWidth={84}
-			smallHeight={0}
-			mediumWidth={84}
-			mediumHeight={0}
-			largeWidth={84}
-			largeHeight={0}
-			breakpoints={getProperties(arcSite)?.breakpoints}
 			resizerURL={getProperties(arcSite)?.resizerURL}
 			resizedImageOptions={resizedImageOptions}
```

Comment:
The image component doesn't take in resizedImageOptions. If you're working on using resized images, I'd recommend looking at the image components storybook https://61eeeb27215b5f003acdf18f-ntxngzkadz.chromatic.com/?path=/docs/components-image--default-story

Reply:
Thanks jack, will work on this

Reply:
@JackHowa : The author service doesn't upload images through photo center and they handle the images themselves so they currently don't generate auth tokens for resizer and we can't generate those ourselves at this time so the author images can't be resized at this time.

Reply:
oh ok. that makes sense. resizedImageOptions can still be removed

Reply:
Yes Jack, Removed it [e750aa1](https://github.com/WPMedia/arc-themes-blocks/pull/1474/commits/e750aa162b00893502fa45533c54c2c1e79f8c47)

Thanks @nschubach @ecarlisle

Reply:
thank you -- addressed